### ISSUE TITLE
Add explicit ordering for .feature files

### ISSUE THREAD

**Issue by benjaoming** [learningequality/kolibri#4583]
### Observed behavior

While working on #4544, we identified a need to sort the .feature files. The discussion mentioned a lot of different ideas, but we've decided to withdraw all of that. The concern is that too much meta data will make the test scenarios hard to maintain or inaccurate.

Currently, our setup contains a directory for each role. The solution drafted:

* add an `ORDER.txt` file for every directory which states the explicit order.
* Parse `ORDER.txt` such that each line is assumed to be a file name.
* If the file does not exist, raise an exception.
* If the file exist, add it to the list of files for this role in the order that it appeared in
* If a line starts with `#`, skip it.
* Ignore empty lines
* If a file is not mentioned in `ORDER.txt` but exists in the filesystem, fail - but allow files to exist in a commented out version (i.e. with `#` starting the line)

Syntax example:

```
admin-change-facility-settings.feature
admin-create-class.feature
admin-create-users.feature
admin-delete-class.feature
admin-delete-user.feature
admin-edit-user.feature
admin-import-content.feature
admin-manage-channel-sizes.feature
admin-change-profile-information.feature
admin-enroll-assign-users.feature
admin-export-data.feature

# This line is ignored and the below file is disabled
# admin-find-user.feature

admin-sign-in.feature
admin-sign-out.feature
admin-change-language.feature
```

### Expected behavior

Generate a spreadsheet for integration testing wit explicit order

### User-facing consequences

n/a

### Errors and logs
n/a
### Steps to reproduce
n/a

### Context
n/a

**Comment by mrpau-richard** [comment]
Thanks for explaining this @benjaoming. I'll start working on it.

**Comment by mrpau-richard** [comment]
Closing this, fixed in #4544
